### ISSUE TITLE
onSend is not triggered

### ISSUE THREAD

**Issue by serolgames** [loresclement/react-native-customizable-chat#1]
Hello,

If I have a custom send button, the onSend event is never triggered. 

```
            <CustomizableChat 
                messages={messages}
                onSend={(e) => sendMessage(e)}
                backgroundColor='transparent'
                debug={false}
                inputPlaceholderValue={t('Votre message...')}
                customSendButton={<Button 
                                    title={t('Envoyer')}
                                />}
            />
```

**Comment by loresclement** [comment]
Hey,

Fixed in 0.3.5

**Comment by loresclement** [comment]
Hey,

Fixed in 0.3.5
